Tests/test_file_tiff.py

```diff
@@ -450,6 +450,18 @@ def test_save_tiff_with_jpegtables(self):
         # Should not raise UnicodeDecodeError or anything else
         im.save(outfile)
 
+    def test_open_tiff_uint16_multiband(self):
+        """Test opening multiband TIFFs and reading all channels."""
+        base_value = 4660
+        for i in range(2, 6):
+            infile = "Tests/images/uint16_{}_{}.tif".format(i, base_value)
+            im = Image.open(infile)
+            im.load()
+            for j in range(i):
+                current_value = base_value + j
+                im.seek(j)
+                self.assertEqual(im.getpixel((0, 0)), current_value)
```

Comment:
I think I was confusing the GDAL API and Pillow's as GDAL uses `seek()` for TIFFs to move between bands. It looks like Pillow doesn't do the same.

So if I instead use the test:
```

    def test_open_tiff_uint16_multiband(self):
        """Test opening multiband TIFFs and reading all channels."""
        base_value = 4660
        for i in range(2, 6):
            infile = "Tests/images/uint16_{}_{}.tif".format(i, base_value)
            im = Image.open(infile)
            im.load()
            print(im.getpixel((0,0)))
            print(im.split())
        self.assertEqual(True, False) # So that the test will print stdout & stderr
```
then I see the following as output:
```
4660
(<PIL.Image.Image image mode=I;16 size=10x10 at 0xAD5BA73E48>,)
4660
(<PIL.Image.Image image mode=I;16 size=10x10 at 0xAD5BA73CC0>,)
4660
(<PIL.Image.Image image mode=I;16 size=10x10 at 0xAD5B921CF8>,)
4660
(<PIL.Image.Image image mode=I;16 size=10x10 at 0xAD5B921F60>,)
```

This this leads me to think that these multiband images are actually only loading as single-channel TIFFs. Is that correct?